### ISSUE TITLE
No .rpm build for Fedora 22/23 

### ISSUE THREAD

**Issue by heyakyra** [atten/zodiac#5]
is this project dead?

**Comment by atten** [comment]
No, it is suspended. I'll continue to develop it soon. Thanks for your interest!

**Comment by Zerokami** [comment]
The project has a Linux binary. So it works with almost all Linux distros. Just make the "zodiac" binary executable and run it.

Creating and testing RPM, DEB,  every thing else makes things complicated. (This is just my personal opinion as a user. The dev may have a different opinion).
